fcrepo-export/src/main/java/org/fcrepo/exporter/Exporter.java

```diff
@@ -147,7 +149,10 @@ void writeResponse(final FcrepoResponse response, final File file)
 
         final List<URI> describedby = response.getLinkHeaders("describedby");
         for (final Iterator<URI> it = describedby.iterator(); describedby != null && it.hasNext(); ) {
-            exportDescription(it.next());
+            exportDescription(response.getUrl(), it.next());
+            if (it.hasNext()) {
+                throw new RuntimeException("fcrepo does not produce multiple describedby headers.");
```

Comment:
Ok, but I *realy* *really* *really* don't want our export format to be tied to the contentious hack that is "fcr:metadata".  Many of us had the understanding that fcr:metadata *is* an RDF representation of the binary.  Could I perhaps serialize fcr:metadata in the way I am, but serialize any other (not-currently-possible) descrbedBy resources as if they were normal resources?